chains/solana/contracts/tests/testutils/anchor.go

```diff
@@ -49,7 +49,7 @@ func FundAccounts(ctx context.Context, accounts []solana.PrivateKey, solanaGoCli
 		remaining = unconfirmedTxCount
 
 		time.Sleep(500 * time.Millisecond)
-		if count > 60 {
+		if count > 120 {
```

Comment:
CI flakiness was very annoying, so increasing the timeout here as we've discussed in today's standup